hoodie-cli/src/main/java/com/uber/hoodie/cli/commands/CommitsCommand.java

```diff
@@ -107,9 +104,9 @@ public String rollbackCommit(
         final String commitTime,
         @CliOption(key = {"sparkProperties"}, help = "Spark Properites File Path")
         final String sparkPropertiesPath) throws Exception {
-        if (!HoodieCLI.tableMetadata.getAllCommits().contains(commitTime)) {
+        if (!HoodieCLI.tableMetadata.getActiveCommitTimeline().containsInstant(commitTime)) {
             return "Commit " + commitTime + " not found in Commits " + HoodieCLI.tableMetadata
-                .getAllCommits();
+                .getActiveCommitTimeline().getInstants().collect(Collectors.toList());
```

Comment:
I am not sure what this helper class will be and how to generalize it. Not doing this one for now.